### PR TITLE
update the crds

### PR DESCRIPTION
**Pull request by william-xiang**

### CODE REVIEW

File: docs/guides/platform/CheCluster_crd_v1.md [hunk 1]
```diff
@@ -45,4 +47,49 @@
...
+### status
+
+**Description:** Defines the observed state of CheCluster.
+
+**Type:**  object
+
+| Property            | Type        | Description                       |
+|---------------------|-------------|-----------------------------------|
+|status.chePhase|string|Specifies the current phase of the Che deployment.|
+|status.chePhase|string|Public URL of the Che server.|
```
Comment (SanjulaGanepola) [inline docs/guides/platform/CheCluster_crd_v1.md]: Are all the properties in this table supposed to have the same property name `status.chePhase`?

File: docs/guides/platform/CheCluster_crd_v1.md [hunk 2]
```diff
@@ -45,4 +47,49 @@
...
+|status.chePhase|string|Currently installed Che version.|
+|status.chePhase|string|The public URL of the internal devfile registry.|
+|status.chePhase|string|Specifies the current phase of the gateway deployment.|
+|status.chePhase|string|A human readable message indicating details about why the Che deployment is in the current phase.|
+|status.chePhase|string|The public URL of the internal plug-in registry.|
+|status.chePhase|string|A brief CamelCase message indicating details about why the Che deployment is in the current phase.|
+|status.chePhase|string|The resolved workspace base domain. This is either the copy of the explicitly defined property of the same name in the spec or, if it is undefined in the spec and we're running on OpenShift, the automatically resolved basedomain for routes.|
```
Comment (SanjulaGanepola) [inline docs/guides/platform/CheCluster_crd_v1.md]: base domain or basedomain?

File: docs/guides/platform/DevWorkspace_operator_CRDs.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+|config.routing.defaultRoutingClass |string       |DefaultRoutingClass specifies the routingClass to be used when a DevWorkspace specifies an empty `.spec.routingClass`. Supported routingClasses can be defined in other controllers. If not specified, the default value of "basic" is used.|
+|config.routing.proxyConfig         |object       |ProxyConfig defines the proxy settings that should be used for all DevWorkspaces. These values are propagated to workspace containers as environment variables.. On OpenShift, the operator automatically reads values from the \"cluster\" proxies.config.openshift.io object and this value only needs to be set to override those defaults. Values for httpProxy and httpsProxy override the cluster configuration directly. Entries for noProxy are merged with the noProxy values in the cluster configuration. To ignore automatically read values from the cluster, set values in fields to the empty string (\"\"). Changes to the proxy configuration are detected by the DevWorkspace Operator and propagated to DevWorkspaces. However, changing the proxy configuration for the DevWorkspace Operator itself requires restarting the controller deployment.|
```
Comment (SanjulaGanepola) [inline docs/guides/platform/DevWorkspace_operator_CRDs.md]: Remove extra period in description of `config.routing.proxyConfig`

File: docs/guides/platform/DevWorkspace_operator_CRDs.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+|config.routing.proxyConfig.httpProxy|string      |HttpProxy is the URL of the proxy for HTTP requests, in the format http://USERNAME:PASSWORD@SERVER:PORT/. To ignore automatically detected proxy settings for the cluster, set this field to an empty string ("").|
+|config.routing.proxyConfig.httpsProxy|string     |HttpsProxy is the URL of the proxy for HTTPS requests, in the format http://USERNAME:PASSWORD@SERVER:PORT/. To ignore automatically detected proxy settings for the cluster, set this field to an empty string ("").|
```
Comment (SanjulaGanepola) [inline docs/guides/platform/DevWorkspace_operator_CRDs.md]: Not very familiar with this configuration stuff, but is `http://USERNAME:PASSWORD@SERVER:PORT/` supposed to use `https://`... for `config.routing.proxyConfig.httpsProxy` or is it intended to be the same format as the previous row?

**Review by SanjulaGanepola — CHANGES_REQUESTED** [review]

File: docs/guides/platform/CheCluster_crd_v1.md (hunk 1 above)
Comment (william-xiang) [reply docs/guides/platform/CheCluster_crd_v1.md]: Good catch. The property names are different. Changes are made in new commit.

File: docs/guides/platform/CheCluster_crd_v1.md (hunk 2 above)
Comment (william-xiang) [reply docs/guides/platform/CheCluster_crd_v1.md]: It should be base domain.

File: docs/guides/platform/DevWorkspace_operator_CRDs.md (hunk 3 above)
Comment (william-xiang) [reply docs/guides/platform/DevWorkspace_operator_CRDs.md]: Fixed in new commit.

File: docs/guides/platform/DevWorkspace_operator_CRDs.md (hunk 4 above)
Comment (william-xiang) [reply docs/guides/platform/DevWorkspace_operator_CRDs.md]: It should be https.

File: docs/guides/platform/DevWorkspace_operator_CRDs.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+**Type:**  object
+
+| Property                          | Type        | Description                       |
+|-----------------------------------|-------------|-----------------------------------|
+|spec                             |object       |Defines the desired state of DevWorkspaceTemplate.|
+|spec.attributes|object|Map of implementation-dependant free-form YAML attributes.|
+|spec.commands|array|Predefined, ready-to-use, devworkspace-related commands|
+|spec.components|array|List of the devworkspace components, such as editor and plugins, user-provided containers, or other types of components|
+|spec.dependentProjects|array|Additional projects related to the main project in the devfile, contianing names and sources locations|
```
Comment (SanjulaGanepola) [inline docs/guides/platform/DevWorkspace_operator_CRDs.md]: contianing -> containing

**Review by SanjulaGanepola — CHANGES_REQUESTED** [review]

File: docs/guides/platform/DevWorkspace_operator_CRDs.md (hunk 5 above)
Comment (william-xiang) [reply docs/guides/platform/DevWorkspace_operator_CRDs.md]: Done.

**Review by SanjulaGanepola — APPROVED** [review]

**Review by eric-simpson — APPROVED** [review]

**Pull request merged**